reference/tax_settings.v3.yml

```diff
@@ -170,6 +170,13 @@ components:
           enum:
             - ON_PAYMENT_CAPTURE
             - ON_ORDER_CREATION
+        rounding_strategy:
+          default: RATE_ROUNDING
+          type: string
+          description: This setting affects the rounding behaviour of tax amounts calculated by the basic tax provider. The NO_ROUNDING option offers a high accuracy total tax amount for orders. The RATE_ROUNDING option means every resulting tax rate amount has been rounded. Note that tax is always calculated on the line item amount after discounts.
+          enum:
+            - NO_ROUNDING
+            - RATE_ROUNDING
```

Comment:
Note: intentionally leaving off LINE_ROUNDING at this time, since we don't have any demand for it.